lib/publiccloud/utils.pm

```diff
@@ -859,4 +860,70 @@ sub detect_worker_ip {
     die "Worker IP could not be determined - return was $ip";
 }
 
+
+
+=head2 trup_call_remote 
+
+    trup_call_remote(command, [timeout, exit_code, proceed_on_failure])
+
+    Run a transactional command on remote machine via ssh, non-interactive
+
+    Return: 
+    ($out, $res): a string with the full screen output and the number of final exit code: 
+
+=cut
+
+sub trup_call_remote {
+    my ($self, $cmd, %args) = @_;
+    my $start = time();
+    my $script = "sudo transactional-update -n $cmd";
+    my $timeout = $args{timeout} // 600;
+    my $exit = $args{exit_code} // 0;
+    my $proceed = $args{proceed_on_failure} // 0;
```

Comment:
same here I hardly can imagine situation with the test where we tolerate migration failure and also what we would do in such case ? 🤔